build.boot

```diff
@@ -219,8 +219,11 @@ Type \"Yes\" to confirm."
                          (clojure.java.io/file)
                          (slurp)
                          (clojure.edn/read-string))
-        config-settings (load-file
-                         (str (System/getProperty "user.dir") "/config/credentials.edn"))]
+        config-settings (or
+                          (load-file
```

Comment:
I would swap the order the files are loaded so that if you have the file in both locations it reverts to `~/.wombats/config.edn`